src/services/document-service.js

```diff
@@ -33,7 +34,8 @@ const searchResultsProjection = {
   slug: 1,
   contributors: 1,
   updatedOn: 1,
-  tags: 1
+  tags: 1,
+  archived: 1
```

Comment:
I would not include this in the search projection, it's not needed.

Reply:
I thought we said we'll want to show for admins the documents that are archived with some sort of labelling (can't really test this yet though, waiting to get your PR merged first)

Reply:
Ah, okay, I didn't know that